src/components/Modal/Modal.mocks.tsx

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ReactElement } from 'react'
+import { action } from '@storybook/addon-actions'
+
+import { Hierarchy, Type } from '../Button/Button.types'
+import { columns, data, rowKey } from '../Table/Table.mocks'
+import { Button } from '../Button'
+import { Modal } from '.'
+import { ModalProps } from './Modal.props'
+import { Table } from '../Table'
+import { useModal } from '../../hooks/useModal'
+
+const { Neutral } = Hierarchy
+const { Outlined } = Type
+
+export const children = (
+  <>
+    {'Modal Content'}
+    <Table
+      columns={columns}
+      data={data.slice(0, 2)}
+      rowKey={rowKey}
+    />
+  </>
+)
+
+export const childrenFullWidth = (
+  <div
+    style={{
+      backgroundColor: '#1e1e1e',
+      color: '#ce9178',
+      fontFamily: 'monospace',
+      height: '100%',
+      padding: 24,
+    }}
+  >
+    Code Block
+  </div>
+)
+
+export const childrenLong = (
+  <>
+    <Table
+      columns={columns}
+      data={data}
+      rowKey={rowKey}
+    />
+    {`Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque vel dolor ac sem congue ornare rhoncus quis tortor.
```

Comment:
Should we move this string to a more generic utils file? So every component can use it

Reply:
Let's rather introduce an ad-hoc library to generate lorem ipsum texts (I'd address this in a dedicated PR) 🤙🏼